mailpoet/lib/Automation/Engine/Data/WorkflowStatistics.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Automation\Engine\Data;
+
+class WorkflowStatistics {
+
+
+  private $workflowId;
+  private $versionId;
+  private $entered;
+  private $inProgress;
+
+  public function __construct(
+    int $workflowId,
+    int $entered = 0,
+    int $inProcess = 0,
+    ?int $versionId = null
+  ) {
+    $this->workflowId = $workflowId;
+    $this->entered = $entered;
+    $this->inProgress = $inProcess;
+    $this->versionId = $versionId;
+  }
+
+  public function getWorkflowId(): int {
+    return $this->workflowId;
+  }
+
+  public function getVersionId(): ?int {
+    return $this->versionId;
+  }
+
+  public function getEntered(): int {
+    return $this->entered;
+  }
+
+  public function hasValues(): bool {
+    return $this->entered > 0;
+  }
```

Comment:
> This name seems a little ambiguous

Actually that was by purpose, but I think, we should just drop this for now. The idea was to provide a quite general "Yes, I have some `values`" information. You have seen, how I used it. Now my thinking is: But having it so general, because this entity might evolve means, its meaning will change over time. How it is used currently is bound to total values...

Now, we could start like `hasSubscribers` or `hasTotals` and add more and more `hasX` methods or drop it and introduce specific methods, when they are actually needed.

117a525af